pom.xml

```diff
@@ -281,11 +281,15 @@
           <fail>true</fail>
           <failFast>false</failFast>
           <rules>
-            <dependencyConvergence></dependencyConvergence>
+            <dependencyConvergence>
+              <ignore />
```

Comment:
No, this is not what I meant with that link. I was showing they use tags that close themselves. `ignore` would be a configuration property to go with that particular plugin

Reply:
Ahh alright, so we should use self-closing tags as well? Not sure what we want to do here.

Reply:
You can see the example for the enforcer plugin that we're using on this page: http://maven.apache.org/enforcer/enforcer-rules/dependencyConvergence.html

Reply:
Alright, so we should be using self-closing tags, as per what they're doing in their example.